proto/connectrpc/conformance/v1/service.proto

```diff
@@ -316,43 +318,53 @@ message ConformancePayload {
     ConnectGetInfo connect_get_info = 4;
   }
   message ConnectGetInfo {
+    // The query params observed in the request URL.
     repeated Header query_params = 1;
   }
 }
 
 // An error definition used for specifying a desired error response
 message Error {
+  // The error code.
   int32 code = 1;
```

Comment:
We might want to change to use an enum. The existing YAML files will all still work (since enums can be provided via their numeric value in JSON and YAML). But it would allow us to replace the numbers with human-readable enum names.

WDYT?

Reply:
Cool. I'll add it to this PR. wdyt about enum name?
* `ConnectErrorCode`
* `ErrorCode`
* `Code

Reply:
I think `Code` would be the most clear, since that matches the name used by both connect-go and grpc-go.